### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by kateloschinina**
Update README with the most relevant info.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,65 @@
 # kat-client-proxies
-A library of client proxies for various apis
+A library of client proxies for various APIs for KAT.
 
-## Current proxies
-* myFTClient
-* acquisitionCtxClient
+KAT (Knowledge & administration tools) is a part of ft.com application created for Financial Times B2B clients.
```
Comment (dapenguin) [inline README.md]: Change "is a part of ft.com" to "is an ft.com"

File: README.md [hunk 2]
```diff
@@ -1,32 +1,65 @@
 # kat-client-proxies
-A library of client proxies for various apis
+A library of client proxies for various APIs for KAT.
```
Comment (dapenguin) [inline README.md]: Change "APIs for KAT" to "APIs used by KAT"

File: README.md [hunk 3]
```diff
@@ -1,32 +1,65 @@
...
 USE_MOCK_API=false
 USER_UUID=
 LICENCE_UUID=
 ```
-Where `USER_UUID` is a valid user uuid on FT.com and `LICENCE_UUID` is the uuid of the licence associated with `USER_UUID`
+
+Where `USER_UUID` is a valid user uuid on FT.com and `LICENCE_UUID` is the uuid of the licence associated with `USER_UUID`. Feel free to approach KAT team via `#ft-syndikat` slack channel if you have any questions.
+
+## Deployment
+This component has been created to be included throughout other KAT components.
```
Comment (dapenguin) [inline README.md]: Similar to what we recently spoke about, consider whether "module" would be more appropriate than "component"

**Review by dapenguin — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (kateloschinina) [reply README.md]: 👍

File: README.md (hunk 1 above)
Comment (kateloschinina) [reply README.md]: 👍

File: README.md (hunk 3 above)
Comment (kateloschinina) [reply README.md]: 👍

File: README.md [hunk 4]
```diff
@@ -22,19 +22,19 @@ LICENCE_UUID=
 Where `USER_UUID` is a valid user uuid on FT.com and `LICENCE_UUID` is the uuid of the licence associated with `USER_UUID`. Feel free to approach KAT team via `#ft-syndikat` slack channel if you have any questions.
 
 ## Deployment
-This component has been created to be included throughout other KAT components.
+This module has been created to be included throughout other KAT components.
```
Comment (dapenguin) [inline README.md]: Should probably change the last word from "components" to "modules"

File: README.md [hunk 5]
```diff
@@ -22,19 +22,19 @@ LICENCE_UUID=
...
 
-### How to update a repo that uses the component to the new version
+### How to update a repo that uses the module to the new version
 If you want to update connected components with the latest version, you need to follow the following steps:
```
Comment (dapenguin) [inline README.md]: Feels like this line should also use "modules" instead of "components".

File: README.md [hunk 6]
```diff
@@ -22,19 +22,19 @@ LICENCE_UUID=
...
-1. Create a new repository release on gitHub. Please follow naming convention of previous releases.
+1. Create a new repository release on GitHub. Please follow naming convention of previous releases.
 2. Go to `package.json` file of the component you want to update, and change `"kat-client-proxies"` dependency version to the [newly released one](https://github.com/Financial-Times/kat-client-proxies/releases).
```
Comment (dapenguin) [inline README.md]: Feels like this line should also use "module" instead of "component".

**Review by dapenguin — DISMISSED** [review]

**Review by dapenguin — APPROVED** [review]
👍

**Pull request merged**